web/templates/pages/members.html

```diff
@@ -20,13 +20,13 @@
 			<ul class="about_list">
 			{% for member in members_list %}
 				{% if member.activity_status == 'MC' %}
```

Comment:
This is really bad practice of defining data. I have no way of knowing what 'MC' means. The only way I can know is to go and find appropriate model. Now imagine if we would have much more models than 3, that would be a nightmare.

I think it would be very vise to revise the model and amend this.

Please remember, the most important thing that you should learn at this level is to write readable code.